packages/cozy-intent/src/api/services/NativeMessenger.spec.ts

```diff
@@ -79,6 +91,8 @@ describe('NativeMessenger', () => {
         type: 'string',
         uri: 'string'
       })
-    ).toThrowError(strings.noListenerFound)
+    ).not.toThrow()
+
+    expect(mockDebug).toHaveBeenCalled()
```

Comment:
nit: why not using toHaveBeenCalledWith() ?

Reply:
@trollepierre I preferred to not check the argument in an effort to stay decoupled from the implementation